src/SFA.DAS.EmployerFinance/Data/Scripts/Ad Hoc/SeedFromEmployerAccount.sql

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Can the folder be renamed from "Ad Hoc" to "Manual"? It seems to be a convention across a few of the projects here.

Reply:
Done.